psp/typings.py

```diff
@@ -68,6 +68,11 @@ class Y:
 
     powers: np.ndarray
 
+    def __eq__(self, other) -> bool:
+        # Currently we interpret `np.nan` as "No prediction for this horizon.",
+        # so `equal_nan=True` make sense.
+        return np.array_equal(self.powers, other.powers, equal_nan=True)
+
```

Comment:
not sure I get this, could you expand on the comment and when this is used

Reply:
You made me realize I forgot to add a new test file in the PR. It's necessary for that test. Basically I compare two  `Y` with some `np.nan` values (the flag is because I want `nan`s to be considered equal, but it's not the case by default) in them and now I can simply do `assert y1 == y2`.